src/Students/Students.java

```diff
@@ -0,0 +1,45 @@
+package Students;
+
+public class Students {
+    private static String name;
+    private static int course;
+    private static int grade;
+
+
+    public void setName(String name) {
+        if (name.length() >= 3) {
+            this.name = name;
+        }
+    }
+
+    public static String getName() {
```

Comment:
Getters and setters can not be static logically because these methods on 100% belong to each object